bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/services/EvaluationService.java

```diff
@@ -130,7 +130,7 @@ public void sourceChanged(int sourcePriority, Map sourceValuesByName) {
 				ISources.ACTIVE_WORKBENCH_WINDOW_SHELL_NAME, ISources.ACTIVE_EDITOR_ID_NAME,
 				ISources.ACTIVE_EDITOR_INPUT_NAME, ISources.SHOW_IN_INPUT, ISources.SHOW_IN_SELECTION,
 				ISources.ACTIVE_PART_NAME, ISources.ACTIVE_PART_ID_NAME, ISources.ACTIVE_SITE_NAME,
-				ISources.ACTIVE_CONTEXT_NAME, ISources.ACTIVE_CURRENT_SELECTION_NAME }));
+				ISources.ACTIVE_CONTEXT_NAME, /* ISources.ACTIVE_CURRENT_SELECTION_NAME */ }));
```

Comment:
Why is this comments?

Reply:
It is not clear to me why this IEvaluationContext has to filter those variables at all. So, in order to minimize regressions, I only removed the "selection" variable from the blacklist.

Reply:
Let's just remove it instead of commenting it out.